share/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: force
+
+DIRS=$(sort $(dir $(wildcard */PKGBUILD)))
+
+all: $(DIRS)
+
+$(DIRS): force
+	cd $@ && makepkg --printsrcinfo > .SRCINFO
```

Comment:
Easier than escaping the redirect in the shell script that does the provisioning.